### ISSUE TITLE
Post titles not used in browser tab

### ISSUE THREAD

**Issue by jrbourbeau** [mrocklin/mrocklin.github.io#5]
I noticed that the title of posts isn't used as the name for browser tab, instead the first section header is used. For example, naively I would have expected this post https://matthewrocklin.com/interviews.html to use "How I Conduct an Interview" for the tab name, but instead "I talk about the company and what we're looking for" is used (the [first section header](https://github.com/mrocklin/mrocklin.github.io/blob/cff019299c6ccb4c009f632a749b6f9ea81362c9/interviews.md))

![Screenshot 2023-01-09 at 1 13 46 PM](https://user-images.githubusercontent.com/11656932/211389630-61473c7e-936e-4f7c-af04-4d0ee77260aa.png)

This is a minor point that I've not looked into. Just opening an issue for visibility.

**Comment by jrbourbeau** [comment]
It might be that the `title:` metadata should be moved to an `h1`-level header (I just came across that being done to resolve this issue in another project)

**Comment by mrocklin** [comment]
Yup.  See https://github.com/wpilibsuite/sphinxext-opengraph/issues/94
